test-exports-cf/src/index.ts

```diff
@@ -43,7 +43,7 @@ export default {
         azureOpenAIApiInstanceName: env.AZURE_OPENAI_API_INSTANCE_NAME,
         azureOpenAIApiDeploymentName: env.AZURE_OPENAI_API_DEPLOYMENT_NAME,
         azureOpenAIApiVersion: env.AZURE_OPENAI_API_VERSION,
-      } 
```

Comment:
This is just some trailing whitespace that my editor removed, I can leave it be if you'd prefer — all the same to me.